### PR TITLE
Update how-to-manage-user-profile-info.yml

### PR DESCRIPTION
**Pull request by azarboon**
The following should be clearly stated for brevity:

"For users whose source of authority is Windows Server Active Directory, you must use Windows Server Active Directory to update their identity, contact info, or job info. After you complete your update, you must wait for the next synchronization cycle to complete before you'll see the changes. However, you are able to update their attributes directly in Entra ID so long as you are updating one of Entra ID attributes such as UsageLocation."

### CODE REVIEW

**Comment by Court72** [comment]
@shlipsey3

Can you review the proposed changes? 

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by shlipsey3** [comment]
Hey @azarboon thanks for this suggestion! I like your revisions, but I'm double-checking on that new line you've proposed. Stay tuned!

File: docs/fundamentals/how-to-manage-user-profile-info.yml [hunk 1]
```diff
@@ -91,7 +91,7 @@ procedureSection:
          - **On-premises:** Accounts synced from Windows Server Active Directory include other values not applicable to Microsoft Entra accounts.
 
         > [!NOTE]
-        > You must use Windows Server Active Directory to update the identity, contact info, or job info for users whose source of authority is Windows Server Active Directory. After you complete your update, you must wait for the next synchronization cycle to complete before you'll see the changes.
+        > For users whose source of authority is Windows Server Active Directory, you must use Windows Server Active Directory to update their identity, contact info, or job info. After you complete your update, you must wait for the next synchronization cycle to complete before you'll see the changes. However, you are able to update their attributes directly in Entra ID so long as you are updating one of Entra ID attributes such as UsageLocation.
```
Comment (shlipsey3) [inline docs/fundamentals/how-to-manage-user-profile-info.yml:94]:
```suggestion
        > For users whose source of authority is Windows Server Active Directory, you must use Windows Server Active Directory to update their identity, contact info, or job info. After making updates, you must wait for the next synchronization cycle to complete before the changes take effect. However, you can update their attributes directly in the Microsoft Entra admin center if you are updating Microsoft Entra ID attributes, such as
`UsageLocation`.
```

**Review by shlipsey3 — COMMENTED** [review]
Just some fine-tuning, otherwise this change is good!

**Comment by azarboon** [comment]
@shlipsey3  thank you for suggestion. I just confirmed it. Kindly please sign off this PR.

**Comment by azarboon** [comment]
@shlipsey3  just my 2 cents: could it be because you put `UsageLocation` in the quotation mark?

**Comment by shlipsey3** [comment]
We use those ticks all the time - it must be something with the yml. Can you remove those and see if it builds?

File: docs/fundamentals/how-to-manage-user-profile-info.yml [hunk 2]
```diff
@@ -91,7 +91,8 @@ procedureSection:
          - **On-premises:** Accounts synced from Windows Server Active Directory include other values not applicable to Microsoft Entra accounts.
 
         > [!NOTE]
-        > You must use Windows Server Active Directory to update the identity, contact info, or job info for users whose source of authority is Windows Server Active Directory. After you complete your update, you must wait for the next synchronization cycle to complete before you'll see the changes.
+        > For users whose source of authority is Windows Server Active Directory, you must use Windows Server Active Directory to update their identity, contact info, or job info. After making updates, you must wait for the next synchronization cycle to complete before the changes take effect. However, you can update their attributes directly in the Microsoft Entra admin center if you are updating Microsoft Entra ID attributes, such as
```
Comment (shlipsey3) [inline docs/fundamentals/how-to-manage-user-profile-info.yml:94]:
```suggestion
        > For users whose source of authority is Windows Server Active Directory, you must use Windows Server Active Directory to update their identity, contact info, or job info. After making updates, you must wait for the next synchronization cycle to complete before the changes take effect. However, you can update their attributes directly in the Microsoft Entra admin center if you are updating Microsoft Entra ID attributes, such as UsageLocation.

**Review by shlipsey3 — APPROVED** [review]

**Comment by shlipsey3** [comment]
#please-close

**Comment by shlipsey3** [comment]
#please-open

**Comment by shlipsey3** [comment]
@azarboon I'm going to try and do this on my end. This article was converted from markdown to YML recently as an experiment and I can't tell what's up. Stay tuned...

**Comment by shlipsey3** [comment]
@azarboon Let's change it to just "Usage location" to match the UI. If you make that change it should go through. I'm afraid if I try to suggest another change GitHub will get grumpy again.

If that doesn't work, let me know how you want to proceed. I can do it on my end, but you won't get the contribution. Sorry the formatting broke it!

**Comment by Jak-MS** [comment]
@shlipsey3 - the yml syntax is fixed so validation isn't broken. let us know how you'd like to proceed on this. thanks

**Comment by azarboon** [comment]
@shlipsey3  I changed it to "Usage Location". Kindly please sign off.

Thanks for your concern to make sure that I get the credit; I appreciate it.

**Comment by shlipsey3** [comment]
It wasn't reflecting that change right away - may have been a caching thing on my end. That's why I spun up a new one just in case. Glad it's resolved - thanks for your patience all!

#sign-off

**Pull request merged**